src/commands/init/generateTemplate/index.js

```diff
@@ -73,7 +73,7 @@ function filterFiles(filters) {
 function render(template, data, callback) {
     let rendered;
     try {
-        rendered = Handlebars.compile(template)(data);
+        rendered = Handlebars.compile(template, data)(data);
```

Comment:
I think it's a good time to address it since this PR is related to that, so if you don't mind I'd like to adjust it with that change sometime tomorrow :)